types/rails__ujs/utils/event.d.ts

```diff
@@ -0,0 +1,10 @@
+export function fire(obj: EventTarget, name: string, data: any): boolean;
+
+export function stopEverything(e: Event): void;
+
+export function delegate(
+    element: EventTarget,
+    selector: string,
+    eventType: string,
+    handler: EventListenerOrEventListenerObject,
```

Comment:
```suggestion
    handler: EventListener,
```

Probably `EventListenerObject` is not supported because `handler.call` is executed directly.

https://github.com/rails/rails/blob/v6.0.3.4/actionview/app/assets/javascripts/rails-ujs/utils/event.coffee#L66